content/articles/2018-12-readxl-1-2-0.Rmarkdown

```diff
@@ -0,0 +1,94 @@
+---
+title: 'readxl 1.2.0'
+slug: readxl-1-2-0
+description: > 
+  readxl 1.2.0 is now on CRAN.
+date: '2018-12-20'
+author: Jenny Bryan
+photo:
+  url: https://twitter.com/Thoughtfulnz/status/987900521309614080
+  author: David Hood
+categories:
+  - package
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+## doesn't seem to be working? using n_max below for now
+options(tibble.print_max = 3)
+```
+
+We're stoked to announce that [readxl](http://readxl.tidyverse.org) 1.2.0 is now available on CRAN. Learn more about readxl at <http://readxl.tidyverse.org>. Detailed notes are always in the [change log](https://readxl.tidyverse.org/news/index.html#readxl-1-2-0).
+
+The readxl package makes it easy to get tabular data out of Excel files and into R with code, not mouse clicks. It supports both the legacy `.xls` format and the modern XML-based `.xlsx` format. readxl is expressly designed to be easy to install and use on all operating systems. Therefore it has no external dependencies, such as Java or Perl, which have historically been a source of aggravation with some R packages that read Excel files.
+
+The easiest way to install the latest version from CRAN is to install the whole tidyverse.
+
+```{r eval = FALSE}
+install.packages("tidyverse")
+```
+
+Alternatively, install just readxl from CRAN:
+
+```{r eval = FALSE}
+install.packages("readxl")
+```
+
+Regardless, you will still need to attach readxl explicitly. It is not a core tidyverse package, i.e. readxl is NOT attached via `library(tidyverse)`. Instead, do this in your script:
+
+```{r}
+library(readxl)
+```
+
+## Column name repair
+
+The most exciting change in the v1.2.0 release is the introduction of the new `.name_repair` argument to `read_excel()`, `read_xlsx()`, and `read_xls()`. readxl exposes the `.name_repair` argument that is [coming soon to version 2.0.0 of the tibble package](https://www.tidyverse.org/articles/2018/11/tibble-2.0.0-pre-announce/). **Note: the following examples were executed with the not-yet-released version 2.0.0 of the tibble package.**
+
+First, rest assured that if your sheet has unique column names, readxl leaves them alone, as always:
+
+```{r}
+read_excel(readxl_example("datasets.xlsx"), sheet = "iris", n_max = 3)
+```
+
+However, spreadsheet column names frequently leave much to be desired. `.name_repair` is a more flexible alternative to passing a specific vector of `col_names`. You can express what you want in two main ways:
+
+  * Levels of name repair:
+    - `"minimal"`
```

Comment:
Worth adding minimal definition (names exist, not `NULL`)?

Reply:
I decided to go very light on such details here, since they appear in full precise form in what I consider to be more "primary documents", i.e. tibble resources.